components/arm/server.go

```diff
@@ -57,7 +63,11 @@ func (s *serviceServer) GetJointPositions(
 	if err != nil {
 		return nil, err
 	}
-	convertedPos := &pb.JointPositions{Values: pos.Values}
+	// Have a default empty joint postions in case the position returned is nil.
```

Comment:
```suggestion
	// Have a default empty joint position objets in case the position returned is nil, this guards against nil objects being transferred over he wire.
```